### ISSUE TITLE
Newsletter Sign-up and Resources - Machine Learning Techniques

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#9685]
Newsletter Sign-up and Resources - Machine Learning Techniques<br>
<br>
https://ift.tt/r45mBc7<br>
<br>
<br>
<br>
<div><div>
<section>	<div> <div>
<div>
<figure>	<img data-orig-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?fit=2267%2C1582&amp;ssl=1" data-lazy-sizes="(max-width: 2267px) 100vw, 2267px" data-orig-size="2267,1582" data-medium-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?fit=200%2C140&amp;ssl=1" data-attachment-id="3665" data-image-meta='{"aperture":"0","credit":"","camera":"","caption":"","created_timestamp":"0","copyright":"","focal_length":"0","iso":"0","shutter_speed":"0","title":"","orientation":"0"}' src="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?fit=2267%2C1582&amp;ssl=1" data-image-description="" data-permalink="https://mltechniques.com/resources/rh4_contours/" data-large-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?fit=400%2C279&amp;ssl=1" data-image-title="RH4_contours" data-lazy-src="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?fit=2267%2C1582&amp;ssl=1&amp;is-pending-load=1" data-lazy-srcset="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?w=2267&amp;ssl=1 2267w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=200%2C140&amp;ssl=1 200w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=400%2C279&amp;ssl=1 400w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=768%2C536&amp;ssl=1 768w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=1536%2C1072&amp;ssl=1 1536w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=2048%2C1429&amp;ssl=1 2048w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=1200%2C837&amp;ssl=1 1200w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=1320%2C921&amp;ssl=1 1320w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?resize=600%2C419&amp;ssl=1 600w, https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/03/RH4_contours.png?w=1920&amp;ssl=1 1920w" width="2267" data-image-caption="" class="wp-post-image" data-comments-opened="1" height="1582">	</figure>	<article>
<p>The following content (books, articles, spreadsheets, source code, data sets, tutorials) also includes material available to subscribers only. For the full book list, see <a href="https://mltechniques.com/shop/">here</a>. For certifications, see <a href="https://mltechniques.com/machine-learning-professional-certifications/">here</a>. I will continue to add a lot more in the coming months.</p>
<figure class="wp-caption"><img data-orig-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/01/interpol1.png?fit=973%2C744&amp;ssl=1" data-orig-size="973,744" data-medium-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/01/interpol1.png?fit=200%2C153&amp;ssl=1" data-attachment-id="3337" data-image-meta='{"aperture":"0","credit":"","camera":"","caption":"","created_timestamp":"0","copyright":"","focal_length":"0","iso":"0","shutter_speed":"0","title":"","orientation":"0"}' data-image-description="" src="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/01/interpol1.png?resize=489%2C374&amp;ssl=1" data-permalink="https://mltechniques.com/2023/01/14/new-interpolation-methods-for-synthetization-and-prediction/interpol1/" data-large-file="https://i0.wp.com/mltechniques.com/wp-content/uploads/2023/01/interpol1.png?fit=400%2C306&amp;ssl=1" data-image-title="interpol1" width="489" data-lazy-fallback="1" data-image-caption="" data-comments-opened="1" class="wp-image-3337" data-recalc-dims="1" height="374"></figure>
<hr>
<h2 class="wp-block-heading">Newsletter Subscription</h2>
<p>To access subscriber-only content, sign-up to our free monthly newsletter at the bottom of this page. Newsletter subscription is free. You will receive an email with the password to access the material in question, listed on this page. This includes books, research articles, professional spreadsheets, original datasets, and more. You can unsubscribe at any time. The monthly newsletter also features new blog posts, books, courses and other resources added to MLTechniques.com in the last 30 days.</p>
<hr>
<h2 class="wp-block-heading">Books and Certifications</h2>
<p>Blog posts are available to everyone from our home page. Books currently available on our e-Store include:</p>
<ol>
<li><a href="https://mltechniques.com/product/ebook-gentle-introduction-to-chaotic-dynamical-systems/">Gentle Introduction To Chaotic Dynamical Systems</a></li>
<li><a href="https://mltechniques.com/product/ebook-synthetic-data/">Synthetic Data and Generative AI</a></li>
<li><a href="https://mltechniques.com/product/intuitive-machine-learning/">Intuitive Machine Learning and Explainable AI</a></li>
<li><a href="https://mltechniques.com/product/album/">Stochastic Processes and Simulations</a></li>
</ol>
<p>These documents are used as textbooks for our AI and machine learning certifications. To learn more about the certifications, <a href="https://mltechniques.com/machine-learning-professional-certifications/">follow this link</a>.</p>
<hr>
<h2 class="wp-block-heading">Free Technical Articles For Subscribers Only</h2>

<p>To open these PDF documents, use the password provided when signing-up or in any edition of our newsletter. The summary is accessible to everyone. When clicking on a “download” link, you get two options: view or download the file. The download option is much faster: the view option takes time to render all the fonts and pictures on the screen (up to 30 seconds for large documents). If you already have a copy of some of these documents, check out the version. The most recent version is posted here.</p>

<p>The LaTeX source is available to subscribers only, and contains the images, bibliography and the source code (in a zip file) used to produce the document. It allows you to easily copy and paste the Python code, change the style and formatting (such as font size) to your liking, or for easy inclusion in other documents. To access it, click on “Full version”. All the Python code is also on GitHub.</p>

<ol>
<li>Synthesizing Geospatial Data with A Simple NoGAN Technique<br><a href="https://mltblog.com/3q75Khw">Download PDF</a> | <a href="https://mltechniques.com/2023/07/28/synthesizing-geospatial-data-a-simple-nogan-technique/">Read summary</a>. 11 pages, July 2023.</li>

<li>Sound Generation in Python: Turning Your Data into Music<br><a href="https://mltblog.com/3PVgDxi">Download PDF</a> | <a href="https://mltblog.com/3rxcndb">Read summary</a>. 4 pages, July 2023. Not password-protected.</li>

<li>Generative AI: Synthetic Data Vendor Comparison and Benchmarking Best Practices<br><a href="https://mltblog.com/3CRbIWN">Download PDF</a> | <a href="https://mltechniques.com/2023/06/16/generative-ai-synthetic-data-vendor-comparison-and-benchmarking-best-practices/">Read summary</a>. 13 pages, June 2023. Not password-protected.</li>

<li>A Synthetic Stock Exchange Played with Real Money<br><a href="https://mltblog.com/3OdxEC7">Download PDF</a> | <a href="https://mltblog.com/42zfGyd">Read summary</a>. 20 pages, May 2023. Not password-protected.</li>

<li>Generating and Videolizing Agglomerative Processes<br><a href="https://mltblog.com/41f06Hl">Download PDF</a> | <a href="https://mltechniques.com/2023/04/17/generating-and-videolizing-agglomerative-processes/">Read summary</a> | <a href="https://mltblog.com/3KG1EmM">Full version</a>. 8 pages, April 2023.</li>

<li>Massively Speed-Up your Learning Algorithm, with Stochastic Thinning<br><a href="https://mltblog.com/3MpNNUi">Download PDF</a> | <a href="https://mltechniques.com/2023/04/07/massively-speed-up-your-learning-algorithm-with-stochastic-thinning/">Read summary</a> | <a href="https://mltblog.com/419kdq5">Full version</a>. 13 pages, April 2023.</li>

<li>Smart Grid Search for Hyperparameter Tuning<br><a href="https://mltblog.com/3KkNB72">Download PDF</a> | <a href="https://mltechniques.com/2023/03/30/smart-grid-search-case-study-with-hybrid-zeta-geometric-distributions-and-synthetic-data/">Read summary</a> | <a href="https://mltblog.com/3KgTLp0">Full version</a>. 8 pages, March 2023.</li>

<li>Feature Clustering: A Simple Solution to Many Machine Learning Problems<br><a href="https://mltblog.com/423ArlT">Download PDF</a> | <a href="https://mltechniques.com/2023/03/12/feature-clustering-a-simple-solution-to-many-machine-learning-problems/">Read summary</a> | <a href="https://mltblog.com/42iLqrZ">Full version</a>. 6 pages, March 2023.</li>

<li>Data Synthetization: enhanced GANs vs Copulas<br><a href="https://mltblog.com/3ZLOgTW">Download PDF</a> | <a href="https://mltechniques.com/2023/03/08/data-synthetization-enhanced-gans-vs-copulas/">Read summary</a>. 14 pages, March 2023. Not password-protected.</li>

<li>Introduction to Discrete Chaotic Dynamical Systems<br><a href="https://mltblog.com/3Eui2EN">Download PDF</a> | <a href="https://mltechniques.com/2023/02/21/introduction-to-discrete-chaotic-dynamical-systems/">Read summary</a>. 17 pages, February 2023.</li>

<li>Random Walks, Brownian Motions, and Related Stochastic Processes<br><a href="https://mltblog.com/3YnNWuv">Download PDF</a> | <a href="https://mltechniques.com/2023/02/11/random-walks-brownian-motions-and-related-stochastic-processes/">Read summary</a> | <a href="https://mltblog.com/3xdahPx">Full version</a>. 15 pages, February 2023.</li>

<li>Math-free, Parameter-free Gradient Descent in Python<br><a href="https://mltblog.com/3XZRYJv">Download PDF</a> | <a href="https://mltechniques.com/2023/01/28/math-free-parameter-free-gradient-descent-in-python/">Read summary</a> | <a href="https://mltblog.com/3HCecv3">Full version</a>. 14 pages, January 2023.</li>

<li>New Interpolation Method for Data Synthetization and Prediction<br><a href="https://mltblog.com/3Xhm9v9">Download PDF</a> | <a href="https://mltechniques.com/2023/01/14/new-interpolation-methods-for-synthetization-and-prediction/">Read summary</a> | <a href="https://mltblog.com/3iAiGca">Full version</a>. 18 pages, January 2023.</li>

<li>Military-grade Fast Random Number Generator Based on Quadratic Irrationals<br><a href="https://mltblog.com/3FkIHU2">Download PDF</a> | <a href="https://mltblog.com/3UPYGPz">Read summary</a>. 5 pages, December 2022. Not password-protected.</li>

<li>Empirical Optimization with Divergent Fixed Point Algorithm – When All Else Fails<br><a href="https://mltblog.com/3HgOFru">Download PDF</a> | <a href="https://mltblog.com/3UCVLtC">Read summary</a>. 12 pages, December 2022. Not password-protected.</li>

<li>Spectacular Videos: Synthetic Universes, with Star Collision Graph<br><a href="https://mltblog.com/3TRxwY4">Download PDF</a> | <a href="https://mltblog.com/3OkDWxI">Read summary</a> | <a href="https://mltblog.com/3UTNyly">Full version</a>. 13 pages, November 2022.</li>

<li>Dynamic Clouds and Landscape Generation: Morphing and Evolutionary Processes<br><a href="https://mltblog.com/3T73vDp">Download PDF</a> | <a href="https://mltechniques.com/2022/11/05/dynamic-clouds-and-landscape-generation-morphing-and-evolutionary-processes/">Read summary</a> | <a href="https://mltblog.com/3sZrbPh">Full version</a>. 9 pages, November 2022.</li>

<li>Advanced Machine Learning with Basic Excel — Simple Alternative to XGBoost.<br><a href="https://mltblog.com/3eIGqIH">Download PDF</a> | <a href="https://mltblog.com/3B4jTxz">Read summary</a> | <a href="https://mltblog.com/3euOdJV">Full version</a>. 10 pages, September 2022.</li>

<li>Machine Learning Cloud Regression: The Swiss Army Knife of Optimization.<br><a href="https://mltblog.com/3Al7FjM">Download PDF</a> | <a href="https://mltblog.com/3Khub1z">Read summary</a> | <a href="https://mltblog.com/3wuuDDU">Full version</a>. 25 pages, August 2022.</li>

<li>Weird Random Walks: Synthetizing, Testing and Leveraging Quasi-randomness.<br><a href="https://mltblog.com/3daQ71F">Download PDF</a> | <a href="https://mltblog.com/3K0YJod">Read summary</a> | <a href="https://mltblog.com/3Qzy5Vh">Full version</a>. 10 pages, August 2022.</li>

<li>Detecting Subtle Departures from Randomness.<br><a href="https://mltblog.com/3z2cHRp">Download PDF</a> | <a href="https://mltechniques.com/2022/07/23/detecting-subtle-departures-from-randomness/">Read summary</a> | <a href="https://mltblog.com/3zvh74V">Full version</a>. 14 pages, July 2022.</li>

<li>Classification and Clustering via Image Convolution Filters — Alternative to Generative Mixture Models.<br><a href="https://mltblog.com/3y3P37Q">Download PDF</a> | <a href="https://mltechniques.com/2022/06/19/fast-classification-and-clustering-via-image-convolution-filters/">Read summary</a> | <a href="https://mltblog.com/3OAlXm9">Full version</a>. 15 pages, June 2022.</li>

<li>The Art of Visualizing High Dimensional Data.<br><a href="https://mltblog.com/3mznsF5">Download PDF</a> | <a href="https://mltechniques.com/2022/06/09/the-art-of-visualizing-high-dimensional-data/">Read summary</a> | <a href="https://mltblog.com/3tRcSgK">Full version</a>. 17 pages [22.8 MB], June 2022.</li>

<li>Gentle Introduction to Linear Algebra, with Spectacular Applications.<br><a href="https://mltblog.com/3M1nfVv">Download PDF</a> | <a href="https://mltechniques.com/2022/05/31/gentle-introduction-to-linear-algebra-with-spectacular-applications/">Read summary</a> | <a href="https://mltblog.com/3n4nW6h">Full version</a>. 9 pages, June 2022.</li>

<li>Interpretable Machine Learning: Multipurpose, Model-free, Math-free Fuzzy Regression.<br><a href="https://mltblog.com/3MOMewc">Download PDF</a> | <a href="https://mltechniques.com/2022/05/22/fuzzy-regression-a-generic-model-free-math-free-machine-learning-technique/">Read summary</a> | <a href="https://mltblog.com/3xNAfbZ">Full version</a>. 11 pages, May 2022.</li>

<li>Classification of Shapes via Explainable AI.<br><a href="https://mltblog.com/3LaD900">Download PDF</a> | <a href="https://mltechniques.com/2022/04/20/computer-vision-shape-classification-via-explainable-ai/">Read summary</a> | <a href="https://mltblog.com/3y93TKb">Full version</a>. 7 pages, April 2022.</li>

<li>Interpretable Machine Learning on Synthetic Data, and Little Known Secrets About Linear Regression.<br><a href="https://mltblog.com/3y4TD67">Download PDF</a> | <a href="https://mltechniques.com/2022/05/07/little-known-secrets-about-interpretable-machine-learning-on-synthetic-data/">Read summary</a> | <a href="https://mltblog.com/3zVnIGr">Full version</a>. 13 pages, May 2022.</li>

<li>New Perspective on the Riemann Hypothesis.<br><a href="https://mltblog.com/3vks5I0">Download PDF</a> | <a href="https://mltechniques.com/2022/07/29/new-perspective-on-the-riemann-hypothesis/">Read summary</a> | <a href="https://mltblog.com/3zg0Uiz">Full version</a>. 23 pages, July 2022.</li>
</ol>

<hr> </article> 
</div>
</div>
  </div> 	</section>
</div></div><br>
<br>
<br>
<br>
<br>
<br>
via Machine Learning Techniques - Machine Learning, Artificial Intelligence, Experimental Math https://ift.tt/nB4VSQs<br>
<br>
July 31, 2023 at 04:49PM
